dashboard/config/application.rb

```diff
@@ -22,7 +23,8 @@ module Dashboard
   class Application < Rails::Application
 
     config.middleware.insert_after Rails::Rack::Logger, VarnishEnvironment
-    config.middleware.insert_after VarnishEnvironment, ChannelsApi
+    config.middleware.insert_after VarnishEnvironment, AssetsApi
+    config.middleware.insert_after AssetsApi, ChannelsApi
```

Comment:
The more of these Sinatra-app 'middleware' layers we append to the Rails application, the more I'd like us to eventually remove them from the middleware stack and instead add them as routed Rack application endpoints, which seems like a more appropriate fit since these Sinatra layers do not gain anything from being chained together and invoked on every single request that goes through our Rails application. (See the Rails-guide section on [Routing to Rack Applications](http://guides.rubyonrails.org/routing.html#routing-to-rack-applications) for the alternative approach I have in mind.)